packages/core/src/schema/models/apiKeys.ts

```diff
@@ -19,7 +19,7 @@ export const apiKeys = latitudeSchema.table(
     token: uuid('token')
       .notNull()
       .unique()
-      .$defaultFn(() => crypto.randomUUID()),
+      .default(sql`gen_random_uuid()`),
```

Comment:
Why this change?

Reply:
This is generated natively by postgreSQL

Reply:
better to let the db handle defaults when possible instead of our code.